src/components/data-processing/Classifier/Classifier.jsx

```diff
@@ -39,28 +41,76 @@ const Classifier = (props) => {
 
   const dispatch = useDispatch();
 
+  const [selectedPlot, setSelectedPlot] = useState('kneePlot');
+  const [plot, setPlot] = useState(null);
+
   const debounceSave = useCallback(
     _.debounce((uuid) => dispatch(savePlotConfig(experimentId, uuid)), 2000), [],
   );
 
   const updatePlotWithChanges = (obj) => {
-    dispatch(updatePlotConfig(plotUuid, obj));
-    debounceSave(plotUuid);
+    dispatch(updatePlotConfig(plots[selectedPlot].plotUuid, obj));
+    debounceSave(plots[selectedPlot].plotUuid);
   };
 
-  const config = useSelector((state) => state.componentConfig[plotUuid]?.config);
+  const plots = {
+    kneePlot: {
+      title: 'Knee Plot',
+      plotUuid: generateDataProcessingPlotUuid(sampleId, filterName, 1),
+      plotType: 'classifierKneePlot',
+      plot: (config, plotData, actions) => {
+
+        return(
+        <ClassifierKneePlot
+          experimentId={experimentId}
+          config={config}
+          expConfig={expConfig}
+          plotData={plotData}
+          actions={actions}
+        />)
+      },
+    },
+    histogram: {
+      title: 'Histogram',
+      plotUuid: generateDataProcessingPlotUuid(sampleId, filterName, 0),
+      plotType: 'cellSizeDistributionHistogram',
+      plot: (config, plotData, actions) => (
+        <ClassifierEmptyDropsPlot
+          experimentId={experimentId}
+          config={config}
+          expConfig={expConfig}
+          plotData={plotData}
+          actions={actions}
+        />
+      ),
+    },
+  };
+
+  const config = useSelector(
```

Comment:
You should put this declaration at the top with the rest of them

Reply:
I think alex did his changes based on CellSizeDistribution.jsx which also has things arranged in this way, I'll change both.

Reply:
Turns out we need them here because they reference `plots`, if we don't have them in this order they won't render correctly.

However, I feel we should refactor `plots` in all components so that they can be set outside of the component and kept as constants, we could create a ticket for that.

Reply:
Yeah good idea